AUTHORS.md

```diff
@@ -13,3 +13,4 @@ Please feel free to add your name to this list if you make a PR
 * Benjamin Bach (benjaoming)
 * Jose Rodriguez (jredrejo)
```

Comment:
Blaine, if you don't mind fixing this:Someone wrote my name incorrectly, It's José Redrejo (or José Redrejo Rodríguez, but not José Rodríguez :) )

Reply:
@jredrejo Will do!